pipeline/scripts/hmdb_part.R

```diff
@@ -13,93 +13,113 @@ outdir <- cmd_args[1]
 scanmode <- cmd_args[2]
 db <- cmd_args[3]
 ppm <- as.numeric(cmd_args[4])
+stdrun  <- cmd_args[5] # "yes"
+std_loc <- cmd_args[6] # "/hpc/dbg_mz/development/HMDB_stukken/DIMS/hmdb_preparts/" 
```

Comment:
What happens if `stdrun == 'no'` and you do not add the path to the hdmb_parts?

Reply:
If stdrun (now standard_run) == 'no', the path to the hmdb_parts is still passed to the config script, see line
file_con = file(paste(tmp_dir, "settings.config", sep = "/")
Is it preferable to take paste0("hmdb_parts_dir=", config$hmdb_parts_dir) out of the list and replace it with:
if (standard_run == "yes") { parameters <- c(parameters, paste0("hmdb_parts_dir=", config$hmdb_parts_dir)) }
?

Reply:
This is related to my question about `db <- cmd_args[3]`. You have two scenarios:
1. 'standard_run' needs: out_dir, scanmode, hmdb_parts_dir
2. 'else run' needs: out_dir, scanmode, db, ppm

To me it does not make sense to provide a hmdb_parts_dir if you do not have a standard run and the same for a standard run where you still not provide db and ppm. 

In python you can solve this by using a sub parser, but I am not sure what the possibilities are in R. Or you can do as you suggest use an if statement.

```R
outdir <- cmd_args[1]
scanmode <- cmd_args[2]
run_type <- cmd_args[3]     -> this will prepare your script for more run types, you can also keep the standard_run yes/no pattern.

if (run_type == "standard"){
  hmdb_parts_dir <- cmd_args[4]
  # skip generating HMDB parts
  ...
} else {
  db <- cmd_args[4]
  ppm <- as.numeric(cmd_args[5])
  # generate HMDB parts in case of non-standard mz range
  ...
}
```

Reply:
Above comments are partly solved, we decided to leave it for now.